powersimdata/tests/mock_input_data.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import pandas as pd
+
+from powersimdata.input.grid import Grid
+
+
+class MockInputData:
+    RESOURCES = {
+        "wind": {"wind", "wind_offshore"},
+        "solar": {"solar"},
+        "hydro": {"hydro"},
+    }
+
+    def __init__(
+        self,
+        grid: Grid,
+        start_time="2016-01-01 00:00:00",
+        periods=24,
```

Comment:
Given this mock class can potentially be used in a general sense, maybe we can make this flexible by specifying the `end_time` or we could keep `periods` argument as well just in case people don't want to bother.

Reply:
Done!